messages.js

```diff
@@ -72,70 +51,40 @@ exports.init = function init(sbot) {
 
   return {
     /**
-     * Generate a message linking an existing feed to a meta feed. `previous`
-     * is the previous message on the meta feed in KVT form. `metadata` is an
-     * optional object to be included (object spread) in `msg.value.content`.
-     *
-     * ```js
-     * const msg = sbot.metafeeds.messages.getMsgValAddExisting(metafeedKeys, null, 'main', mainKeys)
-     * ```
+     * Generate opts for "ssb.db.create" to create a message to be posted on
+     * a metafeed linking to a new feed.
+     * Similar to in `deriveFeedKeyFromSeed`, `feedformat` can be either
+     * `bendybutt-v1` for a metafeed or `classic`. `metadata` is an optional
+     * object to be included (object spread) in the message `content`.
      */
-    getMsgValAddExisting(mfKeys, previous, feedpurpose, feedKeys, metadata) {
-      return add(feedpurpose, undefined, previous, feedKeys, mfKeys, metadata)
-    },
-
-    /**
-     * Generate a message to be posted on meta feed linking feed to a meta feed.
-     * Similar to `deriveFeedKeyFromSeed`, `feedformat` can be either
-     * `bendybutt-v1` for a meta feed or `classic`. `metadata` is an optional
-     * object to be included (object spread) in `msg.value.content`.
-     *
-     * ```js
-     * const msg = sbot.metafeeds.messages.getMsgValAddDerived(metafeedKeys, null, 'main', seed, 'classic')
-     * ```
-     */
-    getMsgValAddDerived(
-      mfKeys,
-      previous,
-      feedpurpose,
-      seed,
-      feedformat,
-      metadata
-    ) {
-      if (feedformat !== 'classic' && feedformat !== 'bendybutt-v1') {
-        throw new Error('Unknown feed format: ' + feedformat)
+    optsForAddDerived(mfKeys, feedpurpose, seed, feedFormat, metadata) {
```

Comment:
Couldn't derived also be buttwoo now?

Reply:
Totally fine that this one of just a 1 to 1 of existing functionality

Reply:
Yeah, let's do a PR for buttwoo too, so we can add tests for it